scripts/cards-generator/src/camera/globals.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+FOCAL_LENGTH = 50
+APERTURE = 24
+FOCUS_DISTANCE = 168.60936
```

Comment:
is there a reason for a very specific and precise value here?

Reply:
no, I got a usd camera from somewhere and these values were set like this so I kept them that way. Happy to change them to something else if there's a more reasonable / standard value for some of these

Reply:
Gotcha! Fwiw, here's the camera we publish in a tutorial. It's got a "look at" constraint in it, which I've copied here for completeness, not because you need one. It might be nice to harmonize the wording in the script with the standard attribute names? Mostly I'm thinking sensorSize. The 24 likely corresponds to the horizontalAperture indicated here in the tutorial.

```usda

def Camera "main_cam" (
    kind = "component"
)
{
    float circleOfConfusionSize = 0.0125
    float4[] clippingPlanes = []
    float2 clippingRange = (10, 500000)
    matrix4d constraintTargets:HeadXf = ( (1, 0, 0, 0), (0, 0.999999999997669, 0, 0), (0, 0, 0.999999999997669, 0), (0, 15, 15, 1) ) (
        constraintTargetIdentifier = "HeadXf"
        customData = {
            bool hasPropertyDefinition = 0
        }
    )
    float3[] extentsHint = [(3.40282e+38, 3.40282e+38, 3.40282e+38), (-3.40282e+38, -3.40282e+38, -3.40282e+38)]
    float focalLength = 25.047468
    float focusDistance = 200
    float fStop = 5.6
    float horizontalAperture = 24
    float lensSqueezeRatio = 1
    token projection = "perspective"
    uniform bool stereoscopic = 0
    float verticalAperture = 13.5
}
```